main.lua

```diff
@@ -143,34 +146,63 @@ function love.draw()
 end
 
 function love.keypressed(key, code, isRepeat)
-    Nuklear.keypressed(key, code, isRepeat)
     if not RELEASE and code == CONFIG.debug.key then
         DEBUG = not DEBUG
     end
+
+    if Nuklear.keypressed(key, code, isRepeat) then
+        return
+    end
+
+    State.current():keypressed(key, code, isRepeat)
```

Comment:
I think it will be more maintainable in the future to just check if the current state has the callback in the first place.